app/models/work.rb

```diff
@@ -0,0 +1,32 @@
+class Work < ApplicationRecord
+  has_many :votes #, through: :users
+
+  validates :title, presence: true, uniqueness: true
+  validates :creator, presence: true
+  validates :publication_year, presence: true, numericality: { only_integer: true }, length: { is: 4 }
+  validates :description, presence: true
+
+  def self.category_filter(category)
+    return self.where(category: category)
+  end
+
+  def self.spotlight
+    @works = Work.all
+
+    max_voted = @works.max_by { |work| work.votes.count}
+    return max_voted
+  end
+
+  def self.top10(category)
+    @works = Work.all
+    category_works = @works.category_filter(category)
+
+    sorted = category_works.sort_by { |work| work.votes.count }.reverse
```

Comment:
Minor knitpick:

It's best to let Postgres (the database) do the sorting and filtering if you can.  You can use `.order` and `.limit` to tell Postgres how to sort and filter, even on an associated model.

Your code works fine, but if the database was huge going through Postgres would be a LOT more efficient.

You can read about it [here](https://tomkadwill.com/2017/12/18/counting-or-sorting-by-an-association-in-a-rails-sql-query.html)